aws/v19.1.0/release.yaml

```
@@ -0,0 +1,168 @@
+# Generated with:
+# devctl release create --name 19.1.0 --base 19.0.1 --provider aws --overwrite --app cert-manager@2.24.1 --app cilium@0.11.1 --app observability-bundle@0.7.3 --app vertical-pod-autoscaler@3.5.3 --app node-exporter@1.16.1 --component kubernetes@1.24.15 --component aws-operator@14.21.0 --component cluster-operator@5.8.0 --app vertical-pod-autoscaler@4.0.0
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v19.1.0
+spec:
+  apps:
+  - componentVersion: 1.24.1
+    dependsOn:
+    - vertical-pod-autoscaler-crd
+    name: aws-cloud-controller-manager
+    version: 1.24.1-gs9
+  - componentVersion: 1.21.0
+    dependsOn:
+    - vertical-pod-autoscaler-crd
+    name: aws-ebs-csi-driver
+    version: 2.25.0
+  - dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - vertical-pod-autoscaler-crd
+    name: cert-exporter
+    version: 2.6.0
+  - componentVersion: 1.8.2
+    dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - vertical-pod-autoscaler-crd
+    name: cert-manager
+    version: 2.24.1
+  - name: chart-operator
+    version: 2.35.0
+  - dependsOn:
+    - aws-cloud-controller-manager
+    name: cilium
+    version: 0.11.1
```

```suggestion
    version: 0.11.2
```

to get a small bugfix in that was introduced with https://github.com/giantswarm/cilium-app/pull/82 and fixed on the next patch release